chromium/pages/popup/onion-rules.js

```diff
@@ -0,0 +1,38 @@
+function e(id) {
+return document.getElementById(id);
+}
+  
+  /**
```

Comment:
Please use correct indentation here.

```suggestion
/**
```